### PR TITLE
Cohorts Form Submissions

### PR DESCRIPTION
**Pull request by BrennaManning**
Admin & Staff can input comments and action steps about a cohort as a whole. Saves to database.

### CODE REVIEW

**Comment by BrennaManning** [comment]
Note: It would make more sense for the cohort to default to the page it is on, but all the ways I tried this it ended up saving as undefined even if it appeared to be that value, so fixing that is a to-do, although it shouldn't be crucial.

File: public/javascripts/cohortController.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+var cohortController = function($scope,  $http, $state) {
+  $scope.cohortName = $scope.$parent.currentCohortName;
+  $scope.cohortInfo = $scope.$parent.currentCohortInfo;
+  console.log('cohort name: ' + $scope.cohortName);
+ 
+
+
+//var addCohortEntryController = function($scope, $http, $location) {
```
Comment (davidabrahams) [inline public/javascripts/cohortController.js]: remove comments and console logs

**Comment by davidabrahams** [comment]
Other than cleaning up some console logs and comments, looks perf! :+1: :bus: :100:

**Comment by davidabrahams** [comment]
Fixes #93

**Comment by aconsilvio** [comment]
👍

**Pull request merged**